src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -272,7 +280,6 @@ export class Dropdown extends Disposable {
 			this._treeContainer.style.width = DOM.getContentWidth(this._inputContainer) - 2 + 'px';
 			this._tree.layout(parseInt(this._treeContainer.style.height));
 			this._tree.setInput(new DropdownModel());
-			this._input.validate();
```

Comment:
Why are we removing this? presumably if the allowed values change we should be checking if the current value is satisfying?

Reply:
this actually doesn't matter now, we will skip the validation while the control is in focus